<launch>

  <node ns="octomap" name="$(anon octomap)" pkg="octomap_server" type="octomap_server_node"
        respawn="false" output="screen">
    <remap from="cloud_in"        to="/xtion/depth_registered/points" />
    <param name="~frame_id"       value="map" />
    <param name="~filter_ground"  value="true" />

  </node>

</launch>
